services/bridges/btp/src/cosmos_relay.star

```diff
@@ -0,0 +1,30 @@
+
+cosmvm_cosmos = import_module("github.com/hugobyte/dive/services/cosmvm/cosmos.star")
+ARCHWAY = "github.com/hugobyte/dive/services/cosmvm/static_files/config/"
```

Comment:
can you please specify the variable properly

Reply:
updated